### PR TITLE
Packaging

### PR DESCRIPTION
**Pull request by greg-hellings**
Added setup.py to allow for packaging of files.

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (robled) [inline .idea/vcs.xml]: Is this directory/file intended to be part of this commit?  I noticed that .idea is part of the .gitignore list.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+from setuptools import setup, find_packages
+from os import path
+
+here = path.abspath(path.dirname(__file__))
+with open(path.join(here, 'README.md')) as f:
+    description = f.read()
+
+setup(
+    name='cinch',
+    version='0.1.6',
```
Comment (robled) [inline setup.py:12]: Does this follow [semantic versioning](http://semver.org/)?  It looks like it, but I just want to be sure.

File: .idea/vcs.xml (hunk 1 above)
Comment (greg-hellings) [reply .idea/vcs.xml]: Ah, one of them sneaked in. I'll excise this.

File: setup.py (hunk 2 above)
Comment (greg-hellings) [reply setup.py:12]: I feel like it is. We're declaring it pre-alpha, it's pre-1.0 versioning. We haven't even made a release yet. And I've just done a lot of iterating on what I would call a 0.1 release here. The .6 came up just as I was testing updating and upgrading it in-place. Our first "release" should probably be called 0.2.0 just to keep things sane.

**Review by robled — APPROVED** [review]

**Pull request merged**